### PR TITLE
Add script to spin up a register environment locally

### PR DESCRIPTION
**Pull request by samcrang**
Opinions?

### CODE REVIEW

**Comment by samcrang** [comment]
The `./load-school-data.sh` step doesn't actually work—I'm looking at fixing it now.

**Comment by samcrang** [comment]
This is fixed now.

At the moment it's a bit awkward for someone to access the basic registers in their browser without doing some `Host` header trickery. If we decide this is useful we can make this easier by adding some `A` records pointing to `127.0.0.1` for `{register,field,datatype}.local.openregister.org`.

File: config.docker.register.yaml [hunk 1]
```diff
@@ -33,9 +32,8 @@ externalConfigDirectory: /tmp
 downloadConfigs: true
 
 # can be http: file s3: or classpath: url
-fieldsYamlLocation: classpath://config/fields.yaml
-
-registersYamlLocation: classpath://config/registers.yaml
+fieldsYamlLocation: http://field/records.yaml?page-size=5000
```
Comment (johnollier) [inline config.docker.register.yaml]: If the ENVIRONMENT variable were passed in we should be able to allow the app to read direct from field.alpha.openregister.org etc In dropwizard we should be able to use field.${ENVIRONMENT:alpha}.openregister.org . Could we then dispense with docker-compose-basic?

Comment (samcrang) [inline config.docker.register.yaml]: The point of this thing is that people can spin up and experiment with the whole stack--we want people to have these registers running locally. I'm only pre-loading them with data to make the initial steps less painful.

**Comment by samcrang** [comment]
Also I didn't realise that `school-data.*` was used as part of the conformance tests 😩.

**Pull request merged**